ui/apps/dashboard/src/components/Metrics/StepsThroughput.tsx

```diff
@@ -0,0 +1,30 @@
+import { Info } from '@inngest/components/Info/Info';
+import { NewLink } from '@inngest/components/Link/Link';
+
+export const StepsThroughput = () => {
+  return (
+    <div className="bg-canvasBase border-subtle overflowx-hidden relative flex h-[300px] w-full flex-col rounded-lg p-5">
```

Comment:
```suggestion
    <div className="bg-canvasBase border-subtle overflow-x-hidden relative flex h-[300px] w-full flex-col rounded-lg p-5">
```
Given that this seems really similar to RunsThroughput.tsx it could possible use the same component